kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -351,3 +357,34 @@ class Transport(val options: TransportOptions) {
         return builder
     }
 }
+
+data class SDKErrorDetailInfo(
+    var message: String,
+    var field: String,
+    var code: String,
+    @SerializedName("documentation_url")
+    var documentationUrl: String,
+)
+
+data class SDKErrorInfo(
+    var message: String,
+    var errors: List<SDKErrorDetailInfo>,
+    @SerializedName("documentation_url")
+    var documentationUrl: String,
+)
+
+fun parseSDKError(msg: String) : SDKErrorInfo {
+    val rx = Regex("\\s+Text:\\s+\"(.*)\"")
```

Comment:
Since you'll have quotes inside the captured text, seems safer, or at least more explicit, to anchor the end quote to the end of the string `$`.

Might also look at string literals to reduce the amount of escaping required https://kotlinlang.org/docs/basic-types.html#string-literals

Reply:
Yeah, that's a good idea. It's been working but it may not without that. Thanks!

Reply:
I did have string literals initially but my regex was wrong. I reverted to the normal pattern for testing, but didn't clean it back up.